Running regression tests with pysim backend

I'm trying to run regression tests in my working directory using this command:

```
scripts/run_tests.py --all --verbose -b pysim regression
```

This is the error I'm getting:

```
fixturefunc = <function verilate_model at 0x7fc811c0bba0>, request = <SubRequest 'verilate_model' for <Function test_entrypoint[rv32uc-rvc-pysim-False-True]>>
kwargs = {'request': <SubRequest 'verilate_model' for <Function test_entrypoint[rv32uc-rvc-pysim-False-True]>>, 'worker_id': 'gw0'}

    def call_fixture_func(
        fixturefunc: "_FixtureFunc[FixtureValue]", request: FixtureRequest, kwargs
    ) -> FixtureValue:
        if is_generator(fixturefunc):
            fixturefunc = cast(
                Callable[..., Generator[FixtureValue, None, None]], fixturefunc
            )
            generator = fixturefunc(**kwargs)
            try:
                fixture_result = next(generator)
            except StopIteration:
>               raise ValueError(f"{request.fixturename} did not yield a value") from None
E               ValueError: verilate_model did not yield a value

.venv/lib/python3.11/site-packages/_pytest/fixtures.py:918: ValueError
```

This feels wrong. Why does pysim even care about `verilate_model`? I don't know enough about `pytest` to debug this quickly.

--- a/test/regression/test_regression.py
+++ b/test/regression/test_regression.py
@@ -91,6 +91,8 @@ def verilate_model(worker_id, request: pytest.FixtureRequest):
     distributed, cocotb, mode. It executes a 'SKIP' regression test which verilates the model.
     """
     if request.session.config.getoption("coreblocks_backend") != "cocotb" or worker_id == "master":
+        # pytest expect yield on every path in fixture
+        yield None
         return
 
     lock_path = "_coreblocks_regression.lock"

